srml/contract/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018-2019 Parity Technologies (UK) Ltd.
+// Copyright 2017-2019 Parity Technologies (UK) Ltd.
```

Comment:
does this module actually exist in 2017 ?

Reply:
Good catch, i thought the cp dates are at the repo level when it sounds like we copyright at the module/file level! 

in that case, it only exists as of Sept 2018. will correct thanks 😄